server/src/main/java/com.github.jsappz.lamegame/server/RuntimeOptions.java

```diff
@@ -0,0 +1,9 @@
+package com.github.jsappz.lamegame.server;
+
+public interface RuntimeOptions {
+    String getDbUrl();
+    String getDbUser();
+    String getDbPassword();
+    boolean isDbEnabled();
```

Comment:
would be nice to add comments here about what each var means. especially the last 2